codespell_lib/_codespell.py

```diff
@@ -411,8 +417,20 @@ def ask_for_word_fix(line, wrongword, misspelling, interactivity):
     return misspelling.fix, fix_case(wrongword, misspelling.data)
 
 
+def print_context(lines, index, context_before, context_after):
+    if context_before < 0:
+        return
+    for i in range(index - context_before, index + context_after + 1):
+        if 0 <= i < len(lines):
+            if i == index:
+                print('> %s' % lines[i].rstrip())
+            else:
+                print(': %s' % lines[i].rstrip())
```

Comment:
Perhaps could be slightly more Pythonic?
```
    for i in range(max(index - context_before, 0),
                   min(index + context_after + 1, len(lines))):
        print('%s %s' % ('>' if i == index else ':', lines[i].rstrip())
```

Reply:
You're right. Let me refactor the code.